decentraland/unity-explorer
# feat: sdk ui enter-leave pointer events

Added support for UI `onMouseEnter` and `onMouseEnter` events.

Issue: https://github.com/decentraland/creator-hub/issues/174

Related PR:
* https://github.com/decentraland/js-sdk-toolchain/pull/1040

-------------------

## QA TEST STEPS

1. Download the build from this PR and open it
2. Go to `65,-125` scene
3. Confirm that every time you put the mouse over the "hover area", its background color changes
4. Confirm that the same happens when you move the mouse outside that same area

https://github.com/user-attachments/assets/f7e2bf72-35cf-4405-b8d0-ba2dc9790e1d
